docs/advance-toolchain-documentation.html

```diff
@@ -1254,6 +1296,7 @@
       <li>An initial response will be attempted within 2 business days.</li>
     </ol>
   </li>
+<li>Also, if you have a GitHub account you can open an issue in the <a href="https://github.com/advancetoolchain/advance-toolchain/issues">Advance Toolchain repository</a>.</li>
 </ul>
 
```

Comment:
@rzinsly 
Few lines back, there is a statement
"Customer support for the Advance Toolchain is provided in one of three ways"
So if its the fourth method, we can update three as four or just leave it as
"Customer support for the Advance Toolchain is provided in the following ways"